must-gather/README.md

```diff
@@ -0,0 +1,25 @@
+cluster-logging must-gather
+=================
+
+`cluster-logging-must-gather` is a tool built on top of [OpenShift must-gather](https://github.com/openshift/must-gather)
+that expands its capabilities to gather Openshift Cluster Logging information.
+
+### Usage
+```sh
+oc adm must-gather --image=quay.io/cluster-logging-dev/cluster-logging-must-gather
+```
+
+The command above will create a local directory with a dump of the cluster-logging state.
+Note that this command will only get data related to the cluster-logging part of the OpenShift cluster.
```

Comment:
What does this mean?  you won't get say info about nodes?

Reply:
In general the must-gather plugin image replaces the default must-gather image. Nevertheless, you can ask `oc adm inspect` to get any standard k8s resource, which we do in the entrypoint `gather` script. Same for PVs, etc.